ui/src/kapacitor/components/AlertTabs.tsx

```diff
@@ -435,6 +429,22 @@ class AlertTabs extends PureComponent<Props, State> {
     return _.get(sections, [section, 'elements', '0'], null)
   }
 
+  private get isMultipleConfigsSupported(): boolean {
+    const {configSections} = this.state
+    const hasPagerDuty2: Section = get(
+      configSections,
+      AlertTypes.pagerduty2,
+      undefined
+    )
+    const hasOpsGenie2: Section = get(
+      configSections,
+      AlertTypes.opsgenie2,
+      undefined
+    )
+    // if kapacitor supports pagerduty2 and opsgenie2, its at least v1.5
+    return !_.isUndefined(hasPagerDuty2) && !_.isUndefined(hasOpsGenie2)
```

Comment:
1. would you include a comment that gives some context about this, or consider making this an `||`? because it's either that all are there, or none or there, but progammatically this line implies to me that there are other cases we should consider. and kafka is also not going to be present... so maybe it could be an || between all three, with a comment explaining that this is a hack?
1. and maybe adding a `hasKafka` for exhaustiveness, though not necessary?